cli/command_repository_create.go

```diff
@@ -41,6 +44,8 @@ func (c *commandRepositoryCreate) setup(svc advancedAppServices, parent commandP
 
 	cmd.Flag("block-hash", "Content hash algorithm.").PlaceHolder("ALGO").Default(hashing.DefaultAlgorithm).EnumVar(&c.createBlockHashFormat, hashing.SupportedAlgorithms()...)
 	cmd.Flag("encryption", "Content encryption algorithm.").PlaceHolder("ALGO").Default(encryption.DefaultAlgorithm).EnumVar(&c.createBlockEncryptionFormat, encryption.SupportedAlgorithms(false)...)
+	cmd.Flag("ecc", "Error correction algorithm.").Hidden().PlaceHolder("ALGO").Default(ecc.DefaultAlgorithm).EnumVar(&c.createBlockECCFormat, ecc.SupportedAlgorithms()...)
+	cmd.Flag("ecc-overhead", "How much space overhead can be used for error correction, in percentage. Use 0 to disable ECC.").Hidden().Default("0").IntVar(&c.createBlockECCOverhead)
```

Comment:
can you rename the flag `ecc-overhead-percent` and all fields along with it? withtout the suffix it's not immediately obvious what the units are.